src/Pos/DB/Block.hs

```diff
@@ -247,13 +248,19 @@ deleteData fp = (liftIO $ removeFile fp) `catch` handle
         | isDoesNotExistError e = pure ()
         | otherwise = throwM e
 
+dirDataPath :: MonadDB m => HeaderHash -> m FilePath
+dirDataPath (formatToString hashHexF -> fn) = gitDirDataPath fn
```

Comment:
I think this was discussed in #vatnik (or somewhere else?) and we decided to go with the hash approach, but I don't remember why.